htdocs/contrat/card.php

```diff
@@ -1612,7 +1612,7 @@
 						$productstatic->status_buy = $objp->tobuy;
 						$productstatic->status_batch = $objp->tobatch;
 
-						print '<td>';
+						print '<td width="85%">';
```

Comment:
There is not reason to have td with width forced to a small value to render better than td with automatic rendering.
This may works for on use case, but will we worse for others.

Reply:
Ok, finally it's the line extrafields that changed the display
Class "titlefieldcreate" causes conflict
I've added a parameter to the showOptionals method, to have class 'left' instead of  'titlefieldcreate'